### ISSUE TITLE
Missing feature: tree loads

### ISSUE THREAD

**Issue by frisodam** [Deltares/GEOLib#129]
Although the TreeLoad class exists in the file `loads.py`, I am still unable to instantiate a tree load because it is still an abstract class (see below). Will the tree load feature be added soon? Thanks in advance.
```
TypeError: Can't instantiate abstract class TreeLoad with abstract method to_internal_datastructure
```

**Comment by carloslubbers** [comment]
Should be fixed in version v1.1.0, you can now use DStabilityModel.add_load to add it to the loads.

You may also use the more manual way and add instances of PersistableTree to DStabilityModel.datastructure.loads[0].Trees

**Comment by frisodam** [comment]
Thank you for the quick update! It works as expected now.
